src/main/java/util/FileHelper.java

```diff
@@ -0,0 +1,85 @@
+package util;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * A helper that manages file I/O.
+ */
+public class FileHelper {
+    private static final Logger logger = LogManager.getLogger(FileHelper.class.getName());
+
+    private static final String CHARSET = "UTF-8";
+
+    private final String fileDirectory;
+    private final String fileName;
+
+    /**
+     * Initialises the FileHelper with the directory and the file name
+     * @param fileDirectory the directory that the config file is located in
+     * @param fileName the name of the config file
+     */
+    public FileHelper(String fileDirectory, String fileName) {
```

Comment:
Is this class only for config, or for general purpose? If for general purpose, update the header comments not to have mention of config. Otherwise, update the class name to reflect it.